### DISCUSSION TITLE
Add the ability to duplicate recipes

### DISCUSSION THREAD

**Discussion by stroodle96** [mealie-recipes/mealie#1198] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Add the ability to duplicate a recipe. I regularly import recipes that include steps for a main course and a side.  This is needed so I can easily duplicate the imported recipe and split it into separate recipe for main course and side.
I can not just import it twice because the site I get them from does not perfectly import the recipe and it requires me to manual fix steps and ingredients. It's would be nice to only have to do this once, then duplicate it. 

**(Optional): Suggest A Solution**
Add Duplicate option to the 3 dots menu under "Share". Ideally it would popup and allow you to edit the name of the new duplicate recipe before saving. Or add (1) to the end and keep the same name.

Another solution would be to not implement this at all and just wait until the mealie share links are importable. Then I could create a share link for the recipe and import it again into my own instance.  This solution could work but would not be as nice as a duplicate menu option.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by mrg358** [comment]
Great idea. Looking for the same feature. 

Mealie currently have UI translation, but missing "content translation". Duplicate recipe would help if user wants to create different language versions from the same recipe.

**Comment by xarantolus** [comment]
I really like this feature and will try to implement it

**Comment by xarantolus** [comment]
I added a pull request, some things are not quite ready but the basics are there: #1750
